src/options.ts

```diff
@@ -35,4 +35,14 @@ export interface CapacitorOptions
 
   /** When enabled, all the threads are automatically attached to all logged events on Android */
   attachThreads?: boolean;
+
+  /**
+  * Enables Out of Memory Tracking for iOS and macCatalyst.
+  * See the following link for more information and possible restrictions:
+  * https://docs.sentry.io/platforms/apple/guides/ios/configuration/out-of-memory/
+  *
+  * @default true
+  * */
+  enableOutOfMemoryTracking?: boolean;
```

Comment:
Do we have this on the React Native SDK? I wonder if this is a sign we should have this TypeScript type definition live elsewhere and use it in both capacitor+RN
cc @marandaneto @jennmueng

Reply:
Yes, it's also on React Native
https://github.com/getsentry/sentry-react-native/blob/3f0c1707d86c1b6e28582d1db21926f2ac543f0b/src/js/options.ts#L76-L83

Reply:
a Unified package would not be bad but each platform has its specific options, not sure how much we'd gain with it.
Not a bad idea tho.